frank/default_parameters.json

```diff
@@ -4,7 +4,7 @@
     "save_dir"         : "",
     "save_profile_fit" : true,
     "save_vis_fit"     : true,
-    "save_uvtables"    : false,
+    "save_uvtables"    : true,
```

Comment:
I realize this creates two ~10 MB datafiles if they're .npz, but I expect the vast majority of users will want an output UVTable to compare with CLEAN or something else in CASA, so the default parameters should produce this.

Reply:
These can be computed from the sol object and the uv table. Perhaps its best to add an option to fit.py that generates these files afterwards. E.g. 
Python -m frank.fit -p params --compute_vis_fit sol.obj

Reply:
Or Python -m frank.fit -uv uvtable --compute_vis_fit sol.obj

Reply:
This is also more flexible since you can use it to compute the vis for different UV tables if you want - might be useful for prediction / proposals

Reply:
That's a good idea, an additional option to just generate the uv tables without refitting (even though in most cases the fitting will be very fast). I've made an issue for this and will add the call syntax and function for it in a separate PR. I think 
`python -m frank.fit -uv uvtable --generate_uvtable sol.obj ` 
would be good. 

I do think it's still best to have `save_uvtables : true` in the default parameter file, as the majority of users, especially as they start using frank for the first time, will want the frank UVTable to compare with clean. I'll add to the logging that when this option is true, frank will be saving 2 UVTables whose size is comparable to the input UVTable. The UVTable saving is 
also the last step in the fit, so if a user killed the process at this point, everything else in the fit would already be saved.

The plotting should also have optionally produce a figure that makes: 
- the unconvolved, deprojected 2D sweep; unconvolved, reprojected 2D sweep; unconvolved, reprojected residuals using the residual UVTable (although this would require some additional functionality to make)
- the same but convolved with a beam, if the user passes in the beam
I've opened an issue for this so it can go in a separate PR.

Reply:
OK